### PR TITLE
VIH-6931 More logging

### PR DESCRIPTION
**Pull request by virensareen**
### JIRA link (if applicable) ###
VIH-6931

### Change description ###
Added more logging to the controllers

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by npoxon — DISMISSED** [review]

File: VideoWeb/VideoWeb/Controllers/AudioRecordingController.cs [hunk 1]
```diff
@@ -55,6 +56,7 @@ public async Task<ActionResult> StopAudioRecordingAsync(Guid hearingId)
             try
             {
                 await _videoApiClient.DeleteAudioApplicationAsync(hearingId);
+                _logger.LogTrace($"Audio recording successfully stopped for hearingId: {hearingId}");
```
Comment (levibutler) [inline VideoWeb/VideoWeb/Controllers/AudioRecordingController.cs:59]: shouldnt use string interpolation for logging, it wont add as custom property :)

**Review by levibutler — CHANGES_REQUESTED** [review]
string interpolation changes, also if another active pr gets in first a lot of the logging and test setup would be redundant :)

File: VideoWeb/VideoWeb/Controllers/HealthCheckController.cs [hunk 2]
```diff
@@ -90,7 +96,8 @@ public async Task<IActionResult> HealthAsync()
             {
                 return StatusCode((int) HttpStatusCode.InternalServerError, response);
             }
-
+           
+            _logger.LogTrace("Healthcheck successful.");
```
Comment (levibutler) [inline VideoWeb/VideoWeb/Controllers/HealthCheckController.cs:100]: not needed

File: VideoWeb/VideoWeb/Controllers/UserDataController.cs [hunk 3]
```diff
@@ -46,6 +46,7 @@ public async Task<ActionResult<IList<CourtRoomsAccountResponse>>> GetCourtRoomsA
                 var courtRoomsAccountResponsesMapper = _mapperFactory.Get<IEnumerable<UserResponse>, IEnumerable<string>, List<CourtRoomsAccountResponse>>();
                 var accountList = courtRoomsAccountResponsesMapper.Map(response, query.UserNames);
 
+                _logger.LogTrace($"Court room accounts retrieved successfully");
```
Comment (levibutler) [inline VideoWeb/VideoWeb/Controllers/UserDataController.cs:49]: not needed

File: VideoWeb/VideoWeb/Controllers/VenuesController.cs [hunk 4]
```diff
@@ -38,6 +38,8 @@ public async Task<ActionResult<JudgeNameListResponse>> GetDistinctJudgeNamesAsyn
             try
             {
                 var judges = await _videoApiClient.GetDistinctJudgeNamesAsync();
+
+                _logger.LogTrace($"Judge names retrieved successfully");
```
Comment (levibutler) [inline VideoWeb/VideoWeb/Controllers/VenuesController.cs:42]: not needed

**Review by levibutler — APPROVED** [review]

**Pull request merged**